### DISCUSSION TITLE
Вопросикус

### DISCUSSION THREAD

**Discussion by misha99fr** [MrErtor/generator#7] [Q&A] [answered]
Почему нельзя закрыть Generator с помощью крестика?

**Comment by ghost** [comment, marked as answer]
> Почему нельзя закрыть Generator с помощью крестика?

Добрый вечер. Проект долгое время не обновлялся, поэтому баг с невозможностью закрыть окно программы пока никто не исправил. Совсем недавно я стал новым разработчиком проекта, поэтому в скором времени ждите глобальное обновление проекта от нас 😉
